lib/monitoring/aws-dynamo/DynamoTableGlobalSecondaryIndexMonitoring.ts

```diff
@@ -74,6 +91,35 @@ export class DynamoTableGlobalSecondaryIndexMonitoring extends Monitoring {
       metricFactory.metricThrottledWriteRequestCount();
     this.indexThrottleCountMetric =
       metricFactory.metricThrottledIndexRequestCount();
+
+    const alarmFactory = scope.createAlarmFactory(
+      namingStrategy.resolveAlarmFriendlyName()
+    );
+    this.gsiAlarmFactory = new DynamoAlarmFactory(alarmFactory);
+    this.throttledEventsAnnotations = [];
+
+    for (const disambiguator in props.addReadThrottledEventsCountAlarm) {
+      const alarmProps = props.addReadThrottledEventsCountAlarm[disambiguator];
+      const createdAlarm = this.gsiAlarmFactory.addThrottledEventsAlarm(
+        this.readThrottleCountMetric,
+        CapacityType.READ,
+        alarmProps,
+        disambiguator
+      );
+      this.throttledEventsAnnotations.push(createdAlarm.annotation);
+      this.addAlarm(createdAlarm);
+    }
+    for (const disambiguator in props.addWriteThrottledEventsCountAlarm) {
+      const alarmProps = props.addWriteThrottledEventsCountAlarm[disambiguator];
+      const createdAlarm = this.gsiAlarmFactory.addThrottledEventsAlarm(
+        this.writeThrottleCountMetric,
+        CapacityType.WRITE,
+        alarmProps,
+        disambiguator
+      );
+      this.throttledEventsAnnotations.push(createdAlarm.annotation);
+      this.addAlarm(createdAlarm);
+    }
   }
```

Comment:
Since there's alarms now, you'll need to add something like [this line](https://github.com/cdklabs/cdk-monitoring-constructs/blob/8999ae6cedc3648a82e9b31c0e78e342dbc2bdd7/lib/monitoring/aws-dynamo/DynamoTableMonitoring.ts#L332):

```ts
props.useCreatedAlarms?.consume(this.createdAlarms());
```